sos/__init__.py

```diff
@@ -23,7 +23,7 @@
 
 import gettext
 
-__version__ = "3.2"
+__version__ = "3.3"
```

Comment:
Committing this in the tree doesn't do any harm (although it's arguable that having `__init__.py` in git is a bug itself - it is a build artifact generated from `__init__.py.in` at build time - it's just been in there for so long I never saw a good reason to remove it).

If you're getting builds that don't update it however that is a problem - possibly something missing in the `setup.py` build (the `Makefile` build does do this substitution correctly).